Flick/Group/VotingResultTableViewCell.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  VotingResultTableViewCell.swift
+//  Flick
+//
+//  Created by Haiying W on 1/25/21.
+//  Copyright © 2021 flick. All rights reserved.
+//
+
+import UIKit
+
+class VotingResultTableViewCell: UITableViewCell {
+
+    // MARK: - Private View Vars
+    private let nameLabel = UILabel()
+    private let numberLabel = UILabel()
+    private let numVoteMaybeLabel = UILabel()
+    private let numVoteNoLabel = UILabel()
+    private let numVoteYesLabel = UILabel()
+    private let posterImageView = UIImageView()
+    private let voteMaybeImageView = UIImageView()
+    private let voteNoImageView = UIImageView()
+    private let voteYesImageView = UIImageView()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        selectionStyle = .none
+        backgroundColor = .offWhite
+
+        numberLabel.textColor = .darkBlueGray2
+        numberLabel.font = .systemFont(ofSize: 14)
+        numberLabel.backgroundColor = .lightGray2
+        numberLabel.textAlignment = .center
+        numberLabel.layer.cornerRadius = 12
+        numberLabel.layer.masksToBounds = true
+        contentView.addSubview(numberLabel)
+
+        posterImageView.backgroundColor = .lightGray
+        posterImageView.layer.cornerRadius = 4
+        contentView.addSubview(posterImageView)
+
+        nameLabel.text = "Wonder Women"
+        nameLabel.font = .systemFont(ofSize: 16)
+        contentView.addSubview(nameLabel)
+
+        voteMaybeImageView.image = UIImage(named: "voteMaybeEmoticon")
+        contentView.addSubview(voteMaybeImageView)
+
+        numVoteMaybeLabel.text = "5"
+        numVoteMaybeLabel.font = .systemFont(ofSize: 16)
+        numVoteMaybeLabel.textColor = .mediumGray
+        contentView.addSubview(numVoteMaybeLabel)
+
+        voteNoImageView.image = UIImage(named: "voteNoEmoticon")
```

Comment:
I have the same note about all image views actually but you dont have to if you think it's wonky! just I know we do a lot of things based on appdev standards but that might not even be the best way to do things